web/src/app/users/UserNotificationRuleListDest.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, ReactNode, Suspense } from 'react'
+import { gql, useQuery } from 'urql'
+import {
+  Button,
+  Card,
+  CardHeader,
+  Grid,
+  IconButton,
+  Theme,
+} from '@mui/material'
+import makeStyles from '@mui/styles/makeStyles'
+import { Add, Delete } from '@mui/icons-material'
+import FlatList from '../lists/FlatList'
+import { formatNotificationRule, sortNotificationRules } from './util'
+import UserNotificationRuleDeleteDialog from './UserNotificationRuleDeleteDialog'
+import { styles as globalStyles } from '../styles/materialStyles'
+import UserNotificationRuleCreateDialog from './UserNotificationRuleCreateDialog'
+import { useIsWidthDown } from '../util/useWidth'
+import { ObjectNotFound, GenericError } from '../error-pages'
+import { User } from '../../schema'
+
+const query = gql`
+  query nrList($id: ID!) {
+    user(id: $id) {
+      id
+      contactMethods {
+        id
+      }
+      notificationRules {
+        id
+        delayMinutes
+        contactMethod {
+          id
+          name
+          dest {
+            type
+            values {
+              fieldID
+              value
+              label
+            }
+            displayInfo {
+              text
+              iconURL
+              iconAltText
+              linkURL
+            }
+          }
```

Comment:
We can limit this to just the fields we need, and also remove the unused bits from the stories data for less noise.
```suggestion
        contactMethod {
          id
          name
          dest {
            type
            displayInfo {
              text
              iconAltText
            }
          }
```